sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -4,6 +4,20 @@ var srlog = SIREPO.srlog;
 var srdbg = SIREPO.srdbg;
 
 SIREPO.NUMBER_REGEXP = /^\s*(\-|\+)?(\d+|(\d*(\.\d*)))([eE][+-]?\d+)?\s*$/;
+SIREPO.SAFE_PATH_REGEXP_EXCLUDE_CHARS = [   // array of illegal filename characters
+    '\\\\',  // doubly escaped
```

Comment:
Always a tricky question. You could use closure variables to accomplish the same thing:

```javascript
SIREPO.app.directive('safePath', function() {
    var unsafe_path_chars = '\\/&:+?\'"<>|'.split('');
    ...
```

Ask @moellep what he prefers. :)